index.js

```diff
@@ -1,37 +1,148 @@
-            const speechText = `The current UV Index is ${uvIndex}`
+        const uvIndex = await getUvIndex(uvLocation)
+        const speechText = `The current UV Index is ${uvIndex}. `
+        const riskText = getRiskText(uvIndex)
+
+        return handlerInput.responseBuilder
+            .speak(speechText + riskText)
+            .reprompt(reprompt)
+            .withShouldEndSession(false)
+            .getResponse();
+    }
+}
 
-            return handlerInput.esponseBuilder
-                .speak(speechText)
-                .reprompt(reprompt)
-                .withShouldEndSession(false)
-                .getResponse();
-        })
+const fallbackIntentHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest' &&
+            handlerInput.requestEnvelope.request.intent.name === 'AMAZON.FallbackIntent'
+    },
+    handle(handlerInput) {
+        return handlerInput.responseBuilder
+            .speak('I am not sure I understood you. Do you want to know about current risk from sunburns?')
+            .reprompt('Want to know if you need sunscreen?')
+            .withShouldEndSession(false)
+            .getResponse()
     }
 }
 
-const errorHandler = {
+const cancelAndStopIntentHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest' &&
+            (handlerInput.requestEnvelope.request.intent.name === 'AMAZON.CancelIntent' ||
+                handlerInput.requestEnvelope.request.intent.name === 'AMAZON.StopIntent')
+    },
+    handle(handlerInput) {
+        return handlerInput.responseBuilder
+            .speak('Stay cool!')
+            .withShouldEndSession(true)
+            .getResponse()
+    }
+}
+
+const helpIntentHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'IntentRequest' &&
+            handlerInput.requestEnvelope.request.intent.name === 'AMAZON.HelpIntent'
+    },
+    handle(handlerInput) {
+        return handlerInput.responseBuilder
+            .speak("Burn Alert helps you plan your day without worrying about wether or not you will be at risk of sunburns from UV rays. Try asking 'Do I need sunscreen?' and provide your location for up-to-date data.")
+            .reprompt("Burn Alert helps you plan your day without worrying about wether or not you will be at risk of sunburns from UV rays. Try asking 'Do I need sunscreen?' and provide your location for up-to-date data.")
+            .withShouldEndSession(false)
+            .getResponse()
+    }
+}
+
+const launchRequestHandler = {
+    canHandle(handlerInput) {
+        return handlerInput.requestEnvelope.request.type === 'LaunchRequest'
+    },
+    handle(handlerInput) {
+        return handlerInput.responseBuilder
+            .speak("Welcome to Burn Alert! Try asking how strong the sun is in Boston.")
+            .reprompt('Want to know how strong the sun is today?')
```

Comment:
The user is going to say `Yes` or `No` to this and you'll throw the FallbackIntent instead of handling it.